tests/e2e/utils/prometheus/IAMRolesUtils.py

```diff
@@ -0,0 +1,141 @@
+import subprocess
```

Comment:
Have you taken a look at this file/ thought about extending it? 
https://github.com/awslabs/kubeflow-manifests/blob/main/tests/e2e/utils/aws/iam.py

Also based on https://github.com/awslabs/kubeflow-manifests/pull/290/files#r915221383 we might not need this file.